tests/testsuite/update.rs

```diff
@@ -1891,9 +1910,28 @@ fn update_breaking() {
         .alternative(true)
         .publish();
 
-    p.cargo("update --breaking")
-        .with_status(1)
-        .with_stderr_contains("[ERROR] unexpected argument '--breaking' found")
+    p.cargo("update -Zunstable-options --breaking")
+        .masquerade_as_nightly_cargo(&["update-breaking"])
+        .with_stderr(
+            "\
+[UPDATING] `dummy-registry` index
+[UPGRADING] multiple ^2.0 -> ^3.0
+[UPGRADING] shared ^1.0 -> ^2.0
+[UPDATING] `alternative` index
+[UPGRADING] alternative-1 ^1.0 -> ^2.0
+[UPGRADING] alternative-2 ^1.0 -> ^2.0
+[UPGRADING] incompatible ^1.0 -> ^2.0
```

Comment:
This is neither the original version requirement nor what we are updating it to (neither has `^`)

Reply:
The issue here is that the upgrading messages are logged in `upgrade_manifests`, which does not know the exact dependency syntax in the TOML files. I could move the logging to `write_manifests` (which does handle the TOML) and improve the messages, but then they come after the update messages (from `resolve_ws`), which looks odd.

I could also just drop the upgrading messages, as you've hinted at before. Or I can wipe out any carets in the messages (but I guess that's equally wrong if the manifest actually has carets).

What do you think?

Reply:
Let's keep it for now and see what feedback we get.

While its not the exact version requirement, this also helps in distinguishing it from the update messages (makes it clearer that its *a* version requirement being affected).